### DISCUSSION TITLE
Proposal for Creating an Educational Course on Orchid Panel

### DISCUSSION THREAD

**Discussion by gabrielestefono** [orchidsoftware/platform#2799] [General]
Hello Orchid Community,

I hope this message finds you well. My name is Gabriel Estéfono, and I'm a developer with a keen interest in the Laravel Orchid Admin Panel. I've been exploring your project for a while and am impressed by its capabilities and the value it provides to its users.I am considering creating an educational course about it, that I plan to offer on Udemy (or another learning platform). The course aims to help more developers understand how to effectively use, covering basics to advanced usage, including practical examples and projects.

Before proceeding, I wanted to reach out to ask for your opinion and if you have any concerns or conditions about this initiative. I am committed to respecting the project's license and ensuring proper credit is given for all utilized resources.

Thank you for your time and consideration. I look forward to hearing from you.

Best regards,
Gabriel Estéfono

**Comment by Hesko** [comment]
That's would be so awesome, I would love to check this course after release!

**Comment by agoalofalife** [comment]
Hello Gabriel,
Sorry for informal greeting, but I hope it's not official message. I am a participant of Orchid Community and I think everyone will be grateful for this opportunity. Also, we have a lack of study content in English and it will be a great addition for all of us and potential students and users.
i would like to wish you good luck 👋

**Comment by SadElephant** [comment]
@gabrielestefono, Good afternoon, this is a great idea. I think such a course will also allow more people to get acquainted with Orchid. I wish you success!
